src/analysis.test.js

```diff
@@ -16,7 +16,7 @@ describe(`run simulation`, () => {
     screening_phoneScreenPassed: random.binomial(1, 0.5),
     interview1_passed: random.binomial(1, 0.5),
     interview2_passed: random.binomial(1, 0.3),
-    interview3_passed: random.binomial(1, 0.7),
+    interview3_passed: random.binomial(1, 0.3),
```

Comment:
This was actually on purpose. I'm calibrating the defaults to my own experience but I see the first interview as being a freeform long chat that's got a 50/50 chance of passing (assuming you made it this far of course), the second is a harder tech or panel, and the third is mostly a culture-fit double-check which you really have to screw up to fail.

For different expectations you can just change the value to your own, or even save and reload your own parameters.